drivers/sub8_videoray_m5_thruster/nodes/thruster_driver.py

```diff
@@ -263,7 +267,7 @@ def command_thruster(self, name, force):
             Example names are BLR, FLH, etc.
         '''
         target_port = self.port_dict[name]
-        margin_factor = 0.8
+        margin_factor = 1.0  # Not sure why this would not be anything but one - David + Jason
```

Comment:
If we have agreed that this should be 1.0, then we should probably just remove the variable altogether.